CHANGELOG.md

```diff
@@ -114,6 +114,9 @@
 - Added support for `KonnectExtension`'s `status` `controlPlaneRefs` and `dataPlaneRefs`
   fields.
   [#1297](https://github.com/Kong/gateway-operator/pull/1297)
+- Added `-enforce-config` flag to enforce the configuration of the `ControlPlane`
```

Comment:
```suggestion
- Added `--enforce-config` flag to enforce the configuration of the `ControlPlane`
```

Reply:
We seem to have a mixed approach to this in CHANGELOG. My reasoning for this was that when you run `manager -h` you get flags with single `-`:

```
./bin/manager -h
Usage:
...
  -enable-controller-konnect
        Enable the Konnect controllers.
  -enable-gateway-api-experimental
        Enable the Gateway API experimental features.
  -enable-validating-webhook
        Enable the validating webhook. DEPRECATED: This flag is no-op and will be removed in a future release.
  -enforce-config
        Enforce the configuration on generated in cluster resources. If set to false, the operator will only enforce the configuration when the owner resource spec changes. (default true)
```

Reply:
yep, sounds good. Both ways should work